Tutorial_1b_Moment_calculation.ipynb

```diff
@@ -0,0 +1,860 @@
+{
```

Comment:
<p>and other moment maps. Can we link to the tutorials page and documentation here?</p><p> </p><p><a href="https://spectral-cube.readthedocs.io/en/latest/moments.html" target="_blank">https://spectral-cube.readthedocs.io/en/latest/moments.html</a></p><p> </p><p><a href="https://radio-astro-tools.github.io/tutorials" target="_blank">https://radio-astro-tools.github.io/tutorials/</a></p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Smithsonian/sma-science-tutorials/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='45'/>